frontend/src/App.svelte

```diff
@@ -275,80 +277,146 @@ const displayInfoMessage= () =>{ //After 1 minute of no nodes recieved, a messag
           enableOptimize: true,
         },
       ],
+    },
+  };
+
+  const handle_close_request = () => {
```

Comment:
Could we think of putting all shortcuts methods in their own file to avoid having a huge App.svelte? Or is it too much work?

Reply:
I can put the shortcut methods in a separate file.

Reply:
Afaik, for the keyboard shortcuts, we need to be able to hook it up in the svelte:window to use it on the native window object. This also means calling the methods from that specific svelte:window position. Breaking out the shortcuts means lending a bunch of app specific functionalities to a new Keys component for instance. 

However - this creates an relationship that looks like App <-> Keys where App needs to call the Keys methods and then the Keys methods calls the Apps methods. I just don't see this happening in our current scope.

Reply:
Ok! sounds good =)